src/Search/Search.js

```diff
@@ -9,6 +9,7 @@ const Search = ({
   className,
   onChange,
   defaultValue = '',
+  icon = 'search-small',
```

Comment:
`'search-small'` should be in `defaultProps`.

Default arguments are initialised on every function call (if argument is `undefined`). 
Meanwhile, `defaultProps` are initialised only once.

Reply:
@DmitriVanGuard , changed